src/screens/Embed/Embed.js

```diff
@@ -64,8 +123,9 @@ export default function Embed() {
         <StateHeader
           location={location}
           locationName={locationName}
+          countyName={selectedCounty?.county}
           intervention={intervention}
-          interventions={interventions}
+          interventions={stateInterventions}
```

Comment:
Note that this matches ModelPage, but may not be the behavior we want.
https://github.com/covid-projections/covid-projections/issues/418

Reply:
Fixed.